bin/e2e

```diff
@@ -16,4 +16,5 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
   RAILS_ENV=test E2E=true bin/e2e-setup
 fi
 
+bundle exec rails assets:precompile;
```

Comment:
I moved this here because this is not part of the database setup. Also, this is just for local development of creating E2E tests.